### PR TITLE
Add charmstore log in to user profile view

### PR DESCRIPTION
**Pull request by hatched**
Adds a button into the profile view to give the user the ability to log into the charmstore. This is intentionally unstyled as there is extensive markup and styling changes that need to be done to this view.

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4325/
Test FAILed.

**Comment by hatched** [comment]
#### To QA

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4329/
Test FAILed.

**Comment by hatched** [comment]
Successful run: http://ci.jujugui.org:8080/job/juju-gui/4330/

File: jujugui/static/gui/src/app/store/env/bakery.js [hunk 1]
```diff
@@ -298,9 +348,6 @@ YUI.add('juju-env-bakery', function(Y) {
       _successfulDischarges: function (originalRequest, jsonMacaroon) {
         var prefix = this.macaroonName + '=';
         document.cookie = prefix + btoa(JSON.stringify(jsonMacaroon));
-        window.addEventListener('unload', function() {
```
Comment (fabricematrat) [inline jujugui/static/gui/src/app/store/env/bakery.js]: did we figure out how to do logout in hjc ?

**Comment by fabricematrat** [comment]
:+1:   code  :ok:  no qa

File: jujugui/static/gui/src/app/store/env/bakery.js (hunk 1 above)
Comment (hatched) [inline jujugui/static/gui/src/app/store/env/bakery.js]: We did not. It's on the docket to discuss though.

**Comment by anthonydillon** [comment]
QA issue:
Chrome is blocking the new window as it is treated as a pop up. Could you use a link with `target="_blank"`?

**Comment by anthonydillon** [comment]
QA worked a treat but there is a qa issue around chrome's built in pop up blocker.

**Comment by kadams54** [comment]
Ran into the same popup blocker issue that @anthonydillon mentioned. It's also an issue in Firefox.

**Comment by hatched** [comment]
Thanks for the reviews! As discussed we'll land this as is because the design and markup will change extensively very soon.

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**